tests/test_modeling_clip.py

```diff
@@ -675,7 +675,9 @@ def test_pt_tf_model_equivalence(self):
 
                 tf_checkpoint_path = os.path.join(tmpdirname, "tf_model.h5")
                 tf_model.save_weights(tf_checkpoint_path)
-                pt_model = transformers.load_tf2_checkpoint_in_pytorch_model(pt_model, tf_checkpoint_path)
+                pt_model = transformers.load_tf2_checkpoint_in_pytorch_model(pt_model, tf_checkpoint_path).to(
+                    torch_device
+                )
```

Comment:
```suggestion
                pt_model = transformers.load_tf2_checkpoint_in_pytorch_model(pt_model, tf_checkpoint_path)
                pt_model = pt_model.to(torch_device)
```
I find it easier to read this way when black reformats on several lines.